docs/mkdocs.yml

```diff
@@ -0,0 +1,27 @@
+site_name: "mkdocstrings-vba"
+site_description: "A VBA handler for mkdocstrings."
+site_url: "https://mkdocstrings.github.io/vba"
+repo_url: "https://github.com/mkdocstrings/vba"
+repo_name: "mkdocstrings/vba"
+docs_dir: "src"
+watch:
+  - ../README.md
```

Comment:
Shouldn't it be `README.md`?

Reply:
No, I'm using the `docs` folder as the base dir for the `snippets` plugin.

Reply:
Ah, right :+1: